test/deno/browser.test.js

```diff
@@ -0,0 +1,17 @@
+import assert from 'assert'
+import { Generator } from '@jspm/generator';
+
+const generator = new Generator({
+  env: [
+    "development",
+    "browser",
```

Comment:
If we remove the `"browser"` condition here, does it work then?

Reply:
Nope, still it won't work.